apps/web/src/components/PoolCandidatesTable/PoolCandidatesTable.stories.tsx

```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import { ComponentStory, ComponentMeta } from "@storybook/react";
 
-import { fakePoolCandidates } from "@common/fakeData";
+import { fakePoolCandidates } from "@gc-digital-talent/fake-data";
```

Comment:
Something weird going on in this story.
![image](https://user-images.githubusercontent.com/8978655/219174272-adf20396-55f8-4575-bc4a-4b5abd18d757.png)
It's actually happening all over the place.  I think Storybook is not using the formatting helpers.
![image](https://user-images.githubusercontent.com/8978655/219174786-ee06e097-b1c6-43c4-add1-8e3d73383148.png)

Reply:
Looks like this is resolved. Not sure why there needs to be an empty span though.

## Screenshot
![Screen Shot 2023-02-21 at 12 27 56](https://user-images.githubusercontent.com/3046459/220417266-b2e52433-0d9b-4d43-bfbf-f509ce9547e0.png)

Reply:
My guess would be that we missed a hidden label to the view accessor. We should probably add that in to add some more context to the link! Though, looks like it is an existing issue and not as a result of this PR.

https://github.com/GCTC-NTGC/gc-digital-talent/blob/5c787a10dc609d51fca60e50aae4d5873497c055/apps/web/src/components/PoolCandidatesTable/PoolCandidatesTable.tsx#L216-L226